test/compile_test/instantiate.hpp

```diff
@@ -308,13 +332,26 @@ void instantiate(RealType)
    boost::math::sinc_pi(v1);
    boost::math::sinhc_pi(v1);
    boost::math::asinh(v1);
-   boost::math::acosh(v1);
+   {
+      using std::fabs;
+
+      auto v_special { v1 };
```

Comment:
Why not just set v_special to something specific that's in domain for acosh?  The fabs is unused as well.